File: project_euler/022_Names_Scores/solve.py

if __name__ == "__main__":
    with open("../assets/names.txt", "r") as f:
        data = f.readline().replace("\"", "").split(",")
        data.sort()
        solution = 0
        for pos in range(len(data)):
            score = sum((ord(char) - 64) for char in data[pos])
            solution += ((pos + 1) * score)

    print(solution)

File: project_euler/001_Multiples_of_3_or_5/solve.c
#include <stdio.h>

int main(){
    int i;
    int solution = 0;

    for (i = 0; i < 1000; i++){
        if ((i % 3 == 0) || (i % 5 == 0)) {
            solution += i;
        }
    }

    printf("%d", solution);

    return 0;
}